system/src/system_cloud_internal.cpp

```diff
@@ -95,6 +182,13 @@ int sendApplicationDescription() {
     return 0;
 }
 
+void registerSystemSubscriptions() {
+    if (!Particle.subscribe("particle", systemEventHandler, MY_DEVICES) ||
+            !Particle.subscribe("spark", systemEventHandler, MY_DEVICES)) {
+        LOG(ERROR, "Particle.subscribe() failed");
+    }
+}
```

Comment:
Since there is no error handling other than logging a failure, would it make sense to break these out into separate calls like they were before so `spark` is always attempted to be registered, even if `particle` fails?  We could probably also change the LOG to LOG_DEBUG to keep the file size down.